Sample app to connect from AKS (#61)
Azure-Samples/azure-cache-redis-samples: tutorial/connect-from-aks/ConnectFromAKS/README.md

> # Connect to Azure Cache for Redis from your AKS hosted application

Reviewer: ```suggestion
# Connect to Azure Cache for Redis from an application running in AKS
```

## Overview

> This repo contains code samples for connecting to Azure Cache for Redis from your AKS hosted application using user assigned managed identity or access key.

Reviewer: ```suggestion
This project shows how to connect to Azure Cache for Redis from an AKS hosted application using a Microsoft Entra user-assigned managed identity or an access key.
```

> This project shows how to connect to Azure Cache for Redis from an AKS hosted application using a Microsoft Entra user-assigned managed identity or an access key.

Reviewer: ```suggestion
This project shows how to connect to Azure Cache for Redis from an application running in AKS using a Microsoft Entra workload identity or an access key.
```

> This project shows how to connect to Azure Cache for Redis from an application running in AKS using a managed identity (AKS workload identity) or an access key.

Reviewer: again be careful about saying 'managed identity'. You could just say 'a Microsoft Entra identity'

> See the [accompanying article](https://docs.microsoft.com/azure/azure-cache-for-redis/cache-tutorial-aks-get-started) for detailed instructions on how to configure your AKS cluster and run this sample.

Reviewer: The linked doc talks about a different sample project, and doesn't describe how to configure Entra ID auth (yet)

Author: Yes, the doc changes are also in PR in parallel.
